tools/stats/upload_test_stats.py

```diff
@@ -317,6 +317,14 @@ def init_value(test_case: Dict[str, Any]) -> Dict[str, Any]:
         action="store_true",
         help="If this is being run through circleci",
     )
+    parser.add_argument(
+        "--changed-files",
+        required=True,
+        help="Space delimited list of associated files (ie. changed in the PR), defaults to empty list.",
```

Comment:
Space delimited is a bit problematic since files could have spaces in their names. Could you make this comma delimited instead?